backend/apps/slack/models/message.py

```
@@ -0,0 +1,84 @@
+
+    try:
+      message = Message.objects.get(slack_message_id=slack_message_id, conversation=conversation)
+    except Message.DoesNotExist:
+      message = Message(slack_message_id=slack_message_id, conversation=conversation)
+
+    message.text = data.get("text", "")
+    message.reply_count = data.get("reply_count", 0)
+    message.thread_timestamp = data.get("thread_timestamp")
+    retrieved_timestamp = data.get("timestamp")
+    if retrieved_timestamp:
+      message.timestamp = datetime.fromtimestamp(float(retrieved_timestamp), tz=timezone.utc)
```

Ruff (0.11.9)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)